docker-compose bash issue

ssnukala opened the pull request:
Added instructions to resolve docker-compose bash issue. 

use `ash` instead of `bash`

ssnukala committed:
Added resolution to Docker installation issue

When using docker-compose bash is not working, the fix is to use ash instead. Added documentation to show the option to use ash if bash fails.

ssnukala committed:
Removed formatting

Removed formatting from ash command

Silic0nS0ldier commented:
@ssnukala Could you give some more info into _why_ this is needed please? One of the major points of using Docker is that the environment is stable, so there shouldn't need to be any kind of fallback re the commands (in fact, it indicates a problem that needs to be addressed).

ssnukala committed:
Added instructions to update uf-assets using Docker

Use the following command to install the assets with docker-compose

```
docker-compose run node npm run uf-assets-install
```

lcharette commented on pages/03.installation/01.environment/03.docker/docs.md:
```diff
@@ -29,10 +29,22 @@ the `-d` flag will launch this in the background so you can continue to use the
 
     $ docker exec -it -u www-data userfrosting_php_1 bash -c 'php bakery migrate'
 
+**_docker-compose bash error : if you get an error like this_**
```
This should be in a warning block.

lcharette commented on pages/03.installation/01.environment/03.docker/docs.md:
```diff
@@ -29,10 +29,22 @@ the `-d` flag will launch this in the background so you can continue to use the
+
+    $ docker exec -it -u www-data userfrosting_php_1 ash -c 'php bakery migrate'
+
 You also need to setup the first admin user (again, your container name may be different depending on the name of your root directory):
```
The `$docker_container ` variable from here should be used for a more plug-and-play instruction : 

```
docker_container=$(docker ps -q --filter="ancestor=userfrosting_php")
```

lcharette reviewed (changes requested):
See other comments here : https://github.com/userfrosting/UserFrosting/pull/984#pullrequestreview-242048327
